src/settings/MappingProfiles/MappingProfileDetails/MappingProfileDetails.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+import { noop } from 'lodash';
+
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Col,
+  ExpandAllButton,
+  Headline,
+  KeyValue,
+  Layer,
+  List,
+  MetaSection,
+  MultiColumnList,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+import {
+  FOLIO_RECORD_TYPES,
+  FullScreenView,
+  Preloader,
+} from '@folio/stripes-data-transfer-components';
+
+import { mappingProfileTransformations } from '../MappingProfilesForm/TransformationsField/transformations';
+
+import css from './MappingProfileDetails.css';
+
+const columnWidths = {
+  fieldName: '45%',
+  transformation: '55%',
+};
+const visibleColumns = ['fieldName', 'transformation'];
+const formatter = {
+  fieldName: record => mappingProfileTransformations.find(({ id }) => id === record.fieldId).displayName,
+  transformation: record => record.transformation,
+};
+
+const MappingProfileDetails = props => {
+  const {
+    onCancel,
+    mappingProfile: {
+      hasLoaded,
+      records,
+    },
+  } = props;
+
+  const record = records?.[0] || {};
+
+  const intl = useIntl();
+
+  const columnMapping = {
+    fieldName: intl.formatMessage({ id: 'ui-data-export.mappingProfiles.transformations.fieldName' }),
+    transformation: intl.formatMessage({ id: 'ui-data-export.mappingProfiles.transformations.transformation' }),
+  };
+
+  return (
+    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
+      {contentLabel => (
+        <Layer
+          isOpen
+          contentLabel={contentLabel}
+        >
+          <FullScreenView
+            id="mapping-profile-details"
+            paneTitle={record.name}
+            onCancel={onCancel}
+          >
+            {!hasLoaded
+              ? <Preloader />
+              : (
+                <div className={css.viewMappingProfileContent}>
```

Comment:
Please intent the code here.

Reply:
Lint show error, actually there are all fine

Reply:
Looks good. But I found one more comment. Sorry.